tests/ui/unit/controllers/autoclassify.tests.js

```diff
@@ -1,104 +1,104 @@
 'use strict';
 
-describe('AutoClassify panel', function() {
+// describe('AutoClassify panel', function() {
```

Comment:
I'll leave it up to you on whether it is worth refactoring this to reflect the current state of the code. But if you're just going to comment this out, I would rather remove the file.